.travis.yml

```diff
@@ -26,6 +27,7 @@ script:
 - travis-cargo --only stable test -- --features=deadlock_detection
 - travis-cargo --only beta test -- --features=deadlock_detection
 - travis-cargo --only nightly test -- --all --no-run --target x86_64-fortanix-unknown-sgx
+- travis-cargo --only nightly test -- --all --no-run --target wasm32-unknown-unknown
```

Comment:
I'm aware this does not even test with the atomic `target_future` at the moment... Will have to fix that before this would make sense to have even.